src/main/java/org/graylog/integrations/ipfix/codecs/IpfixCodec.java

```diff
@@ -82,18 +91,54 @@
     protected IpfixCodec(@Assisted Configuration configuration, IpfixAggregator ipfixAggregator) throws IOException {
         super(configuration);
         this.ipfixAggregator = ipfixAggregator;
-        // Standard IPFIX definition file is loaded from the classpath and is mandatory. We don't check for null or empty.
+
         final URL standardIPFixDefTemplate = Resources.getResource(IpfixCodec.class, IPFIX_STANDARD_DEFINITION);
-        final String ipFixCustomDefPath = (String) configuration.getSource().get(CK_IPFIX_DEFINITION_PATH);
-        if (ipFixCustomDefPath == null || ipFixCustomDefPath.trim().isEmpty()) {
+        final List<String> customDefFilePathList = configuration.getList(CK_IPFIX_DEFINITION_PATH);
+        List<URL> filePaths = new ArrayList<>();
+
+        if (customDefFilePathList == null || customDefFilePathList.isEmpty()) {
             infoElementDefs = new InformationElementDefinitions(standardIPFixDefTemplate);
         } else {
-            final URL customIPFixDefURL = Paths.get(ipFixCustomDefPath).toUri().toURL();
-            infoElementDefs = new InformationElementDefinitions(standardIPFixDefTemplate, customIPFixDefURL);
+            errOnInvalidFilePath(customDefFilePathList);
```

Comment:
@danotorrey . I can change it to what you wish for. Also, It would be nice, if I can write something like this below.
      
```
.data(parse-config,ied,ip-fix-codec)
.err(bad-path,overload,interference,security-attack)
```